docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 app:
-  image: unocha/alpine-base-nginx
+  image: unocha/nginx:1.14.2
```

Comment:
The only other material change in the branch. After consulting with @teodorescuserban I was informed we no longer support `latest` tags in dockerhub and this needed to be updated anyway.